examples/example_plugin/example_plugin/filter_extensions.py

```diff
@@ -0,0 +1,48 @@
+from django import forms
+
+from nautobot.extras.plugins import PluginFilterExtension
+from nautobot.utilities.filters import MultiValueCharFilter
+
+
+def suffix_search(queryset, name, value):
+    return queryset.filter(description=f"{value[0]}.nautobot.com")
+
+
+class TenantFilterSetExtension(PluginFilterExtension):
+    model = "tenancy.tenant"
+
+    _fsf_description = MultiValueCharFilter(field_name="description", label="Description")  # Creation of a new filter
+    _fsf_sdescrip = MultiValueCharFilter(
+        field_name="description", label="Description", method=suffix_search
+    )  # Creation of new filter with custom method
+    _fsf_dtype = MultiValueCharFilter(
+        field_name="sites__devices__device_type__slug", label="Device Type"
+    )  # Creation of a nested filter
+
+    filterset_fields = {
+        "example_plugin_description": _fsf_description,
+        "example_plugin_dtype": _fsf_dtype,
+        "example_plugin_sdescrip": _fsf_sdescrip,
+    }
```

Comment:
This feels a bit gross to me. What do you think about declaring the filters/fields inline within the appropriate dict instead as a pattern, something like:

```suggestion
    filterset_fields = {
        # Creation of a new filter
        "example_plugin_description": MultiValueCharFilter(field_name="description", label="Description"),
        # Creation of a nested filter
        "example_plugin_dtype": MultiValueCharFilter(
            field_name="sites__devices__device_type__slug", label="Device Type"
        ),
        # Creation of new filter with custom method
        "example_plugin_sdescrip": MultiValueCharFilter(
            field_name="description", label="Description", method=suffix_search
        ),
    }
```

Reply:
👍 This was a quick transformation from the example. Yes this looks better.